main.go

```diff
@@ -168,6 +168,12 @@ func main() {
 	}
 
 	if *prompt {
+		// DSN strings look like:
+		// user:password@tcp(0.0.0.0:3306)/
+		// With this flag the user can avoid typing their password:
+		// user@tcp(0.0.0.0:3306)/
+		// Save text before : and after @ so we can insert the password
+		// to create a proper DSN string.
```

Comment:
I apologize for being so picky but do you mind also adding the quotes I suggested? :-)